Document new caching behavior in Dart / Kotlin / Swift (#202)
configcat/docs: website/docs/sdk-reference/dart.md

final client = ConfigCatClient.get(sdkKey: '#YOUR-SDK-KEY#');
```

### 4. (Optional) Set up Flutter caching

> If you're using the SDK in a Flutter application, we recommend to use our [Flutter Preferences Cache](https://github.com/configcat/flutter-preferences-cache) for caching. It uses the [shared_preferences](https://pub.dev/packages/shared_preferences) package to store the downloaded `config.json`.

Reviewer: ```suggestion
If you're using the SDK in a Flutter application, it's recommended to use the [Flutter Preferences Cache](https://github.com/configcat/flutter-preferences-cache) implementation for caching. It stores the downloaded `config.json` using the [shared_preferences](https://pub.dev/packages/shared_preferences) package.
```

> ```dart

Reviewer: Isn't an import missing?

Author: Done

final client = ConfigCatClient.get(
    sdkKey: '#YOUR-SDK-KEY#',
    options: ConfigCatOptions(cache: ConfigCatPreferencesCache()));
```

### 5. Get your setting value
```dart

> final isMyAwesomeFeatureEnabled = await client.getValue(key: '<key-of-my-awesome-feature>', defaultValue: false);

Reviewer: key: 'isMyAwesomeFeatureEnabled'

Author: Done

final settingValuesTargeting = await client.getAllValues(user);
```

## Cache

> The SDK caches the downloaded `config.json` only in memory by default. In case you have a Flutter application, you can use our [Flutter Preferences Cache](https://github.com/configcat/flutter-preferences-cache) for caching.  

Reviewer: ```suggestion
The SDK caches the downloaded `config.json` only in memory by default. In case you have a Flutter application, you can use the [Flutter Preferences Cache](https://github.com/configcat/flutter-preferences-cache) for caching.  
```
